doc/manpages/man1/afpldaptest.1.xml

```diff
@@ -89,9 +91,15 @@
     </variablelist>
   </refsect1>
 
-  <refsect1 id="see_also">
+  <refsect1>
     <title>SEE ALSO</title>
 
     <para><citerefentry><refentrytitle>afp.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para>
   </refsect1>
+
+  <refsect1>
+      <title>AUTHOR</title>
+
+      <para>See <ulink url='https://github.com/Netatalk/netatalk/blob/main/CONTRIBUTORS'>https://github.com/Netatalk/netatalk/blob/main/CONTRIBUTORS</ulink></para>
```

Comment:
Oh and a personal taste thing, but I think this would look cleaner:

<para>See <ulink url='https://github.com/Netatalk/netatalk/blob/main/CONTRIBUTORS'>CONTRIBUTORS</ulink></para>

This decouples the human readable text from the underlying repo location. It should signal to someone familiar with an open source project repo structure that we're referring to a standard file that lives in the root of the repo.

Thoughts?

Reply:
Fine by me.